deb.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+tmpdir=build-deb-package
+debfile=egpg.deb
+
+# Make a control file needed to create debian package
+# Contains all information about the package
+mkdir -p $tmpdir/DEBIAN/
+cat <<EOF > $tmpdir/DEBIAN/control
+Source: egpg
+Package: egpg
+Version: 1.0
+Section: custom
+Priority: optional
+Architecture: all
+Depends: bash, gnupg 
```

Comment:
Sorry, I did not notice this previously. There are more dependencies, which are listed on the README: `gnupg2 pinentry-tty haveged libgfshare-bin parcimonie qrencode imagemagick zbar-tools wget realpath psmisc`

Also, there are some build dependencies: 
```
apt install ruby-dev gcc
gem install ronn
```